tests/integration/remote/util.py

```diff
@@ -119,23 +126,32 @@ def do_ocean_tx_and_handle_gotchas(ocean, alice_wallet):
     symbol = random_chars()
 
     print("Call create() from data NFT, and wait for it to complete...")
-    try:
-        priority_fee, max_fee = get_gas_fees_for_remote()
-        data_nft = ocean.data_nft_factory.create(
-            {
-                "from": alice_wallet,
-                "priority_fee": priority_fee,
-                "max_fee": max_fee,
-            },
-            symbol,
-            symbol,
-        )
-        data_nft_symbol = data_nft.symbol()
-    except ERRORS_TO_CATCH as e:
-        if error_is_skippable(str(e)):
-            warnings.warn(UserWarning(f"Warning: EVM reported error: {e}"))
-            return
-        raise (e)
+    num_retries = 2
+    while num_retries != 0:
+        try:
+            priority_fee, max_fee = get_gas_fees_for_remote()
+            data_nft = ocean.data_nft_factory.create(
+                {
+                    "from": alice_wallet,
+                    "priority_fee": priority_fee,
+                    "max_fee": max_fee,
+                },
+                symbol,
+                symbol,
+            )
+            data_nft_symbol = data_nft.symbol()
+            break
+        except ERRORS_TO_CATCH as e:
+            if error_is_skippable(str(e)):
+                warnings.warn(UserWarning(f"Warning: EVM reported error: {e}"))
+                return
+            if "Tx dropped" in str(e):
```

Comment:
is "Tx dropped" not part of the skippable errors?

Reply:
"Tx dropped" is not mentioned in `error_is_skippable`